src/commands/createWebApp/setDefaultRgAndPlanName.ts

```diff
@@ -77,9 +77,8 @@ function checkPlanForPerformanceDrop(asp: AppServicePlan): boolean {
 
 async function promptPerformanceWarning(context: IActionContext, asp: AppServicePlan): Promise<void> {
     context.telemetry.properties.performanceWarning = 'true';
-    const workspaceConfig: WorkspaceConfiguration = workspace.getConfiguration(extensionPrefix);
     const showPlanPerformanceWarningSetting: string = 'showPlanPerformanceWarning';
-    const showPerfWarning: boolean | undefined = workspaceConfig.get(showPlanPerformanceWarningSetting);
+    const showPerfWarning: boolean | undefined = getGlobalSetting(showPlanPerformanceWarningSetting);
```

Comment:
Use `getWorkspaceSetting` here. _We_ set this at the global level, but there's nothing stopping a user from manually editing their settings to disable this for an individual workspace